client/components/Rating.jsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import styled from 'styled-components';
+import avg from '../helpers/avg.js';
+
+
+
+const Grid = styled.div`
+  width: 1000;
+  height: 300px;
+  display: grid;
+  grid-template-columns: 1fr 2fr 1fr 2fr;
+  grid-gap: 2em;
+  grid-auto-rows:75px;
+
+  align-items: center;
+
+  `;
+
+const Nested = styled.div`
+
+  height: 30px;
+
+  font-size: 18px;
+  font-family: Circular, -apple-system, system-ui, Roboto, "Helvetica Neue", sans-serif;
+  letter-spacing: normal;
+
+  align-items: center;
+  justify-items: center;
+
+`;
+
+const Bar = styled.div`
+
+  width: ${props => props.width};
+  background-color: ${props => props.color};
+  height: 10px;
+  position: absolute;
+
+
+`;
+
+const BarCont = styled.div`
+
+  position: relative;
+  margin-top: 10px;
+
+
+
+`;
+ const Wrapper2 = styled.div`
+   display: grid;
+
+grid-template-columns: 25% 75%;
+grid-template-rows: 1fr 1fr;
+
+
+margin: 0 auto;
+
+
+ `;
+ const Span = styled.div`
+ display: grid;
+  justify-items: end;
+
+ `;
+
+
+
+
+
+
+const Ratings = (props) => {
+
+  const findAvg = (item) => {
+    const numbers2 = props.reviews.map(review =>
+      review[item]
+    );
+    const avgs2 = avg(numbers2);
+    return avgs2.toFixed(1);
+
+  }
+
+
+
+
+
+  return (
+
+
+    <Grid>
+    <Nested>Cleanliness</Nested>
```

Comment:
I would consider maybe indenting all the div boxes in between Grid since Grid is the parent div box.